lib/routes/account.js

```diff
@@ -447,15 +518,22 @@ module.exports = function (
           }
         }
 
-        function sendNewDeviceLoginNotification () {
-          if (config.newLoginNotificationEnabled && wantsKeys(request)) {
-            // The response doesn't have to wait for this,
-            // so we don't return the promise.
-            mailer.sendNewDeviceLoginNotification(
-              emailRecord.email,
```

Comment:
You mean if sign-in confirmation is disabled to fallback to new device notifications?